src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import classNames from 'classnames';
+import { FilterParam } from '../../types/FilterParam';
+import { Todo } from '../../types/Todo';
+import {
+  checkCompletedTodo,
+  counterOfActiveTodos,
+  filterTodos,
+} from '../../helpers/helpers';
+
+type Props = {
+  todos: Todo[],
+  filterType: FilterParam,
+  setFilterType: (type: FilterParam) => void,
+  removeTodo: (todoId: number) => void,
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filterType,
+  setFilterType,
+  removeTodo,
+}) => {
+  const handleFilterChange = (event: React.MouseEvent<HTMLAnchorElement>) => {
+    const valueOfSortType = event.currentTarget.text;
+
+    switch (valueOfSortType) {
+      case FilterParam.All:
+        return setFilterType(FilterParam.All);
+
+      case FilterParam.Active:
+        return setFilterType(FilterParam.Active);
+
+      case FilterParam.Completed:
+        return setFilterType(FilterParam.Completed);
+
+      default:
+        return setFilterType(FilterParam.All);
+    }
+  };
+
+  const removeCompletedTodos = () => {
+    if (checkCompletedTodo(todos)) {
+      const completedTodos = filterTodos(todos, FilterParam.Completed);
+
+      completedTodos.map(todo => removeTodo(todo.id));
```

Comment:
Use `forEach` instead of `map` for iterating over `completedTodos` since you are not returning any value from the loop.